x-pack/qa/openldap-tests/src/test/java/org/elasticsearch/xpack/security/authc/ldap/SearchGroupsResolverTests.java

```diff
@@ -217,6 +227,12 @@ protected String bindPassword() {
 
     @Override
     protected String trustPath() {
```

Comment:
This is currently not used anymore because of the way GroupsResolverTestCase is setup. We need to cleanup GroupsResolverTestCase to not rely on class path resources that are outside of a jar. it shouldn't matter. But this rework is out of scope of this PR.